civictechprojects/static/css/partials/_AboutProject.scss

```diff
@@ -215,7 +215,10 @@
     width: 100%;
   }
 }
-
+.AboutProjects-jumplink {
+  /* anchor tag is moved "higher up" than its content to account for Bootstrap fixed nav bar */
```

Comment:
Because our nav bar is fixed, when we have a jumplink to a section on this page it will dutifully scroll so the anchor is at the top of the window - which is then hidden behind the nav. This is a no-JS trick to just let us 'pad' that out a bit, so our jumplinks will work as expected and the content will not be hidden behind the navigation.